module/programs/helper.js

```diff
@@ -389,22 +389,26 @@ module.exports = class ProgramsHelper {
    * @method
    * @name addEntitiesInScope
    * @param {String} programId - Program Id.
-   * @param {Array} entities - entities data.
-   * @param {Object} tenantData - tenantData
+	 * @param {Object} bodyData - body data.
+	 * @param {Object} userDetails - User Details
+	 * @param {Boolean} organizations - If organizations is Present.
    * @returns {JSON} - Added entities data.
    */
 
-  static addEntitiesInScope(programId, entities,tenantData) {
+  static addEntitiesInScope(programId, bodyData,userDetails,organizations) {
```

Comment:
@MallanagoudaB We should consider admin passing key ALL for any entity or organization. that time we should add/ remove this key from entities / organization. if ALL key is passed as a value in organization we can skip validation and add ALL for updation. same for entities  as well. Please make same change in project service as well